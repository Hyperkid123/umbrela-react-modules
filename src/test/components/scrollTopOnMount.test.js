import React from 'react'
import ScrollToTopOnMount from '../../common/components/scrollTopOnMount';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Scroll top on mount component', () => {
  it('shloud return null', () => {
    const tree = shallow(
      <ScrollToTopOnMount/>
    )
    expect(toJson(tree)).toMatchSnapshot();
  })
});
