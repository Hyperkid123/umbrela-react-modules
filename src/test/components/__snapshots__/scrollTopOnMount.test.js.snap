// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll top on mount component shloud return null 1`] = `""`;
